services/121-service/src/metrics/metrics.service.ts

```diff
@@ -107,11 +107,12 @@ export class MetricsService {
             HttpStatus.BAD_REQUEST,
           );
         }
-        return this.getAllPeopleAffectedList(
+        return this.getAllPeopleAffectedList({
```

Comment:
Maybe easy to incorporate in the PR:
```suggestion
        return this.getAllRegistrationsList({
```

But if it requires a lot of 'collateral changes', its maybe cleaner to do a separate search-and-replace-PR.

Reply:
I assume that there is a lot of "PeopleAffected" -> "Registrations" in the backend, so I think it would be cleaner / easier to do them in one fell swoop